.github/ISSUE_TEMPLATE/bundler-related-issue.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bundler related issue
+about: This template is intended for Bundler specific related issues.
+title: 'Bundler: [Please provide a title here]'
+labels: ''
+assignees: ''
+
+---
+
+Thank you for contributing to the [rubygems](https://github.com/rubygems/rubygems) repository, and specifically to the [Bundler](https://bundler.io/) gem.
+
+Before processing your issue please confirm that:
+
+- [ ] You checked all the following documents and couldn't find a solution for your issue:
+
+- [Troubleshooting common issues](https://github.com/rubygems/bundler/blob/master/doc/TROUBLESHOOTING.md)
+- [Bundler documentation site](https://bundler.io/)
+- [Bundler man pages](https://bundler.io/man/bundle.1.html)
+- [Bundler command line reference](https://bundler.io/v1.11/commands.html)
+
+If you haven't done that, please do it before creating a new issue.
+
+Please answer the following questions so we can process your issue as fast as possible:
+
+1. Do you have a reproduction script? It would be really helpful for us if you provide one. You can click [here](https://gist.github.com/xaviershay/6207550) and [here](https://gist.github.com/xaviershay/6295889) too for a couple of examples.
```

Comment:
Perhaps we can avoid "here" as the link text, by editing the sentence.

```suggestion
1. Do you have a reproduction script? It would be really helpful for us if you provide one. Examples: [Sample repro script for Bundler issues](https://gist.github.com/xaviershay/6207550), [Another sample repro script for Bundler issues](https://gist.github.com/xaviershay/6295889).
```